src/main/java/acme/roles/Client.java

```diff
@@ -0,0 +1,49 @@
+
+package acme.roles;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+import org.hibernate.validator.constraints.URL;
+
+import acme.client.data.AbstractRole;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+public class Client extends AbstractRole {
+	// Serialisation identifier -----------------------------------------------
+
+	private static final long	serialVersionUID	= 1L;
+
+	// Attributes -------------------------------------------------------------
+
+	@NotBlank
+	@Column(unique = true)
+	@Pattern(regexp = "CLI-[0-9]{4}")
+	private String				identificatio;
+
+	@NotBlank
+	@Size(max = 75)
+	private String				companyName;
+
+
+	@NotBlank
+	public enum Priority {
```

Comment:
Bad implementation, you define the type priority but you didn't use it as an attribute.